radix-engine/src/vm/wasm/wasmi.rs

```diff
@@ -1305,3 +1309,132 @@ impl WasmEngine for WasmiEngine {
         instance
     }
 }
+
+// Below tests verify WASM "mutable-global" feature, which allows importing/exporting mutable globals.
```

Comment:
I spent a significant amount of time trying to test this feature in rust. But unfortunately Rust does not support it ATM. And yes, referenced issues are pretty old, but haven't found anything more up to date.

Taking above into account I am not really sure if testing it for WASM only makes sense. But we can always disable or remove it.